std.go

```diff
@@ -47,6 +51,28 @@ func (l *stdErrLogger) Errorf(_ context.Context, format string, v ...any) {
 	stdf("ERROR", format, v...)
 }
 
+// AddAttributes adds attributes to include in middleware-driven logs
+func (l *stdErrLogger) AddAttributes(attrbs map[string]any) {
+	l.mu.Lock()
+	if l.attributes == nil {
+		l.attributes = make(map[string]any)
+	}
+
+	for k, v := range attrbs {
+		l.attributes[k] = v
+	}
+	l.mu.Unlock()
```

Comment:
Unless there is a code abstraction reason that makes it not ideal, I would like to use the pattern of deferring the Unlock.

ie:
```
l.mu.Lock()
defer l.mu.Unlock()
```

This makes it easier to audit the code.